src/main/java/net/mcreator/ui/workspace/WorkspacePanelVariables.java

```diff
@@ -125,11 +127,11 @@
 					};
 				} else if (modelColumn == 3) {
 					VariableElementType data = (VariableElementType) elements.getValueAt(row, 1);

```

Comment:
This line causes some issues because it tries to cast a string into a VariableElementType, I think it should be `VariableElementType data = VariableElementTypeLoader.INSTANCE.getVariableTypeFromString((String) elements.getValueAt(row, 1));`

Reply:
You're right. I forgot this one when I fixed the panel.